instrumentation/jetty/jetty-9.1/src/main/java/io/opentelemetry/instrumentation/auto/jetty/package-info.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module provides support for creating server spans for Jetty Handlers.
+ *
+ * <p>It is possible to write web application running on <a
+ * href="https://www.eclipse.org/jetty/">Eclipse Jetty</a> without actually writing any servlets.
+ * Instead one can use <a
+ * href="https://www.eclipse.org/jetty/documentation//current/jetty-handlers.html">Jetty
+ * Handlers</a>.
+ *
+ * <p>As instrumentation points differ between servlet instrumentations and this one, this module
+ * has its own {@code JettyHandlerInstrumentation} and {@code JettyHandlerAdvice}. But this is the
```

Comment:
`JettyHandlerAdvice` -> `Jetty91HandlerAdvice`

Reply:
Missed that - thanks!